pyleoclim/utils/tsutils.py

```diff
@@ -58,25 +63,21 @@ def time_unit_to_datum_exp_dir(time_unit, time_name=None):
         if time_name.lower() == 'age':
             direction = 'retrograde'
     
-    match_a  = ['y', 'yr', 'yrs', 'year', 'years']
-    match_ka = ['ky', 'kyr', 'kyrs', 'kiloyear', 'ka'] 
-    match_Ma = ['ma', 'my','myr','myrs']
-    match_Ga = ['ga', 'gy', 'gyr', 'gyrs']
-    if any(c in time_unit.lower() for c in match_ka):
+    if time_unit.lower() in MATCH_KA:
         datum = 1950
         exponent = 3
         direction = 'retrograde'
-    elif any(c in time_unit.lower() for c in match_a):
+    elif time_unit.lower() in MATCH_A:
         exponent = 0
-    elif any(c in time_unit.lower() for c in ['yr BP', 'yrs BP', 'years BP']):
+    elif time_unit.lower() in ['yr BP', 'yrs BP', 'years BP']:
         datum = 1950
         exponent = 0
         direction = 'retrograde'
-    elif any(c in time_unit.lower() for c in match_Ma) :
+    elif time_unit.lower() in MATCH_MA:
         datum = 1950
         exponent = 6
         direction = 'retrograde'
-    elif any(c in time_unit.lower() for c in match_Ga):
+    elif time_unit.lower() in MATCH_GA:
         datum = 1950
         exponent = 9
         direction = 'retrograde'
```

Comment:
I can't comment on the exact lines, but below here on lines 87-95 there is a separate set of if/else statements. Why aren't those included with the ones above?

Reply:
```
match_a  = ['y', 'yr', 'yrs', 'year', 'years']
    match_ka = ['ky', 'kyr', 'kyrs', 'kiloyear', 'ka'] 
    match_Ma = ['ma', 'my','myr','myrs']
    match_Ga = ['ga', 'gy', 'gyr', 'gyrs']
```
these are statements about the exponent, which usually tell you about the direction and the datum as well. (eg if you are dealing with Ma or Ga, you could not care less whether the datum is 1950 or 2000).  this bit deals with datum/direction:

```
if 'b2k' in tu:
        datum = 2000
        direction = 'retrograde'
    elif any(c in tu for c in ['bp', 'bnf', 'b1950']):
        datum = 1950
        direction = 'retrograde'
    elif any(c in tu for c in ['ad', 'ce']):
        datum = 0
        direction = 'prograde'
```
It is true that both sets of statements partially overlap, so I can see a potential for some conflict if the users provide highly non-standard names, but I think those rules will work in the vast majority of cases. We will likely have to modify them as we encounter more examples, but that's the bane of being a paleoclimatologist, and we can deal with that, so please leave that on our plate.

Reply:
Understood. I was just concerned about the overlap. Happy to leave it to you :)